tests/unit/storage/metadata/mongoclient/MongoClientInterface.spec.js

```diff
@@ -3051,3 +3051,195 @@ describe('MongoClientInterface, deleteBucketIndexes', () => {
         });
     });
 });
+
+describe('MongoClientInterface, putObjectVerCase1', () => {
```

Comment:
should these not go in the other file, which already has a `MongoClientInterface:putObjectVerCase1` test suite?